### PR TITLE
[UI] Reducing the length of label names

### LINKED ISSUE #1934: [UI]: GitHub Quickstart is not mobile friendly
**Describe the bug**

Fields and Descriptions in GitHub Quickstart is not responsiveness.

**To Reproduce**

Steps to reproduce the behaviour:

1. Go to 'http://127.0.0.1:5080/quickstart'
2. Choose <iPhone 8 device
3. See error

**Expected behaviour**

Make the page responsive for all the mobile device.

**Screenshots**

Screenshot 2019-11-09 at 5 16 51 PM

**See the Label Cutout and Also see the extra space in `Description` and the `Field`**

### PR DESCRIPTION
**Pull request by vishakhanihore**
Github Bug/Issue: Fixes #1934

### CODE REVIEW

**Comment by vishakhanihore** [comment]
@djmitche @helfi92 Review Please :)

File: ui/src/views/Quickstart/index.jsx [hunk 1]
```diff
@@ -380,7 +380,7 @@ export default class QuickStart extends Component {
         <Fragment>
           <div className={classes.orgRepo}>
             <TextField
-              label="Organization Name"
+              label="Orga Name"
```
Comment (djmitche) [inline ui/src/views/Quickstart/index.jsx]: I think GitHub uses the short name "Org".

Comment (vishakhanihore) [reply ui/src/views/Quickstart/index.jsx]: Made the change

Comment (vishakhanihore) [reply ui/src/views/Quickstart/index.jsx]: Please review 😄

File: changelog/bug-1934.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+level: silent
+reference: bug 1934
```
Comment (helfi92) [inline changelog/bug-1934.md]:
```suggestion
reference: issue 1934
```

**Comment by vishakhanihore** [comment]
@helfi92

**Comment by vishakhanihore** [comment]
@helfi92 Its good to be merged

**Comment by vishakhanihore** [comment]
@helfi92 done the rebase 😄

**Pull request merged**